src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'point')
+def point_at_obj(robot, bottle, manip=None):
+    '''
+    @param robot The robot performing the point
+    @param bottle The bottle to point
+    @param manip The manipulator to perform the grasp. if None 
+        the active manipulator on the robot is used
+    '''
```

Comment:
Two typos:

- "The bottle to point" -> "The bottle to point at"
- "to perform the grasp" -> "to point with"

Also, I generally see docstrings with double quotes (`"""`) instead of single quotes (`'''`).